src/Doctrine/EventSubscriber/TaskListSubscriber.php

```diff
@@ -49,11 +48,12 @@ public function getSubscribedEvents()
         );
     }
 
-    private function calculate(TaskList $taskList)
+    private function calculate(TaskList $taskList, EntityManagerInterface $em)
     {
         $coordinates = [];
         $tasks = [];
-        $vehicle = $taskList->getVehicle(); 
+        $vehicle = $taskList->getVehicle();
+        $uow = $em->getUnitOfWork();
```

Comment:
it seems you are not using neither `$uow`, nor `EntityManagerInterface $em`

Reply:
cleaned